source/simulator/parameters.cc

```diff
@@ -229,6 +229,22 @@ namespace aspect
                        "The name of the directory into which all output files should be "
                        "placed. This may be an absolute or a relative path.");
 
+    prm.declare_entry ("Binary data directory", "input",
```

Comment:
Instead of "Include directory," I renamed to "Binary data directory" and some extra parameters. Is there a way to specify a parameter keyword in the parameter file that can be replaced by a formatted integer value at run time?